scripts/minikube.sh

```diff
@@ -248,6 +261,7 @@ teardown-rook)
     ${minikube} ssh "sudo mkdir -p /mnt/${DISK}/var/lib/rook; sudo ln -s /mnt/${DISK}/var/lib/rook /var/lib/rook"
     ;;
 cephcsi)
+    validate_container_cmd
```

Comment:
why validation only when copying cephcsi image why not k8s sidecar images?

Reply:
good catch, will fix it.

Reply:
May be I will call `validate_container_cmd` in copy_image_to_cluster() itself